app/code/community/Inviqa/SymfonyContainer/Helper/ContainerProvider.php

```diff
@@ -22,14 +29,15 @@ class Inviqa_SymfonyContainer_Helper_ContainerProvider
      * @var CompilerPassInterface
      */
     private $_storeConfigCompilerPass;
-
     /**
      * @var CompilerPassInterface
      */
     private $_injectableCompilerPass;
 
     public function __construct(array $services = array())
     {
+        $this->_mageApp = isset($services['app']) ? $services['app'] : Mage::app();
```

Comment:
Is the suggestion around replacing `isset(..)` with `instanceof(..)`? What type would `instanceof` check against? The interface `MageApp`? What would be the advantage doing that?

`isset` was used as that looks to be the standard way of checking for injectable dependencies in this project.